programs/mango-v4/src/instructions/serum3_place_order.rs

```diff
@@ -424,22 +424,26 @@ pub fn apply_settle_changes(
     after_quote_vault: u64,
     after_oo: &OpenOrdersSlim,
     health_cache: Option<&mut HealthCache>,
+    fees_to_dao: bool,
```

Comment:
is apply_settle_changes not called in this file at all? why is it in this file then?

Reply:
Only because apply_settle_changes() works via apply_vault_difference() and the latter is only needed in place_order. If you don't feel too strongly about this, I'd leave it as-is.